menas/ui/components/dataset/addDataset.fragment.xml

```diff
@@ -15,7 +15,7 @@
 
 <core:FragmentDefinition xmlns="sap.m" xmlns:core="sap.ui.core" xmlns:form="sap.ui.layout.form"
                          xmlns:hdfs="components.hdfs">
-    <Dialog id="addDatasetDialog" title="{entity>/title} Dataset" stretchOnPhone="true" class="minW40rem">
+    <Dialog id="addDatasetDialog" title="{entity>/title} Dataset" stretchOnPhone="true" class="minW55rem maxW75p w100p">
```

Comment:
Not sure if from here. But the dialogue window is across the whole window now.

![image](https://user-images.githubusercontent.com/17467768/106569214-edfa9e80-653c-11eb-9e23-e6346e8fecd8.png)

Reply:
Yeah should stretch 75% width max. Much easier to read in my opinion.

Reply:
Happy to restyle if you have suggestions :)

Reply:
Alright so I restyled this, made the default width smaller and also optimized for smaller screen. I think this looks better than before and is more usable imho